### DISCUSSION TITLE
What are the BUS addresses for 8086?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#112607] [Programming Help]
### Body

# What are the BUS addresses for 8086? 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by vacisdev** [comment]
The 8086 microprocessor has a time-multiplexed 16-bit address/data bus (AD 15 - AD 0) and a 4-bit address/status bus (A 19 /S 6 - A 16 /S 3).
